tests/system/small/test_pandas.py

```diff
@@ -45,6 +45,56 @@ def test_concat_series(scalars_dfs):
     pd.testing.assert_series_equal(bf_result, pd_result)
 
 
+@pytest.mark.parametrize(
+    ("args"),
```

Comment:
Can directly specify `("prefix', "prefix_sep", "dummy_na", "columns", "drop_first")`

Reply:
Good point, thanks!  I took your other suggestion about using a dictionary, so the tuple isn't needed at all anymore.